tests/cover/test_sampled_from.py

```diff
@@ -17,11 +17,20 @@
 
 from __future__ import division, print_function, absolute_import
 
+import pytest
+
 from hypothesis import given, settings
-from hypothesis.strategies import sampled_from
+from hypothesis.errors import HypothesisDeprecationWarning
+from hypothesis.strategies import sets, integers, sampled_from
```

Comment:
isort has got weird recently. Maybe we should use this as an excuse to change to a more normal sort order.